packages/client/utils/retroGroup/updateClonePosition.ts

```diff
@@ -25,22 +52,56 @@ const getTransition = (isClipped: boolean, timeRemaining: number) => {
 }
 
 export const getDroppingStyles = (
-  targetEl: HTMLDivElement,
+  element: HTMLDivElement,
   bbox: ClientRect,
   maxTop: number,
-  timeRemaining: number
+  timeRemaining: number,
+  targetId?: string | null,
+  isClose?: boolean,
+  lastZIndex?: number
 ) => {
   const spotlightEl = document.getElementById('spotlight')
-  const showAboveSpotlight = spotlightEl ? spotlightEl.contains(targetEl) : false
+  const isTargetInSpotlight = targetId
+    ? !!(spotlightEl && document.querySelector(`div[${DragAttribute.DROPPABLE}='${targetId}']`))
```

Comment:
Before moving to Maintainer Review, I think it'd be best to merge with master and check how things are working. I suspect `isTargetInSpotlight` might be inaccurate as Spotlight results won't always have the droppable attribute [see here](https://github.com/ParabolInc/parabol/blob/7168dcb35410e60c57f8b68719e0490b69eeb35c/packages/client/components/ReflectionGroup/ReflectionGroup.tsx#L208).

Instead, we could follow the pattern in `DraggableReflectionCard` where we query the local Relay store to get the Spotlight result ids. We could copy the lines [here](https://github.com/ParabolInc/parabol/blob/2360f8cb7609ba483b2473c9e7635c204cdf7a85/packages/client/components/ReflectionGroup/DraggableReflectionCard.tsx#L89-L110) and add them to the `useDraggableReflectionCard` hook on line 40. Then, we can pass the `resultIds` to `getDroppingStyles` and see if `resultsIds.includes(targetId)`.

This would mean we're passing 4 extra arguments into `getDroppingStyles` just to get the Spotlight animation. Therefore, it might be cleaner to create a new function called `getSpotlightAnimation` and do all of the calculations there. The function would return `animation`, and we'd pass that to `RemoteReflection` as a prop. This would mean that we wouldn't need to destucture `animation` from `nextStyle` in the `RemoteReflection`.

Reply:
@nickoferrall Thanks! You are right. After rebase, nothing is working as expected. I made changes according to your comment. 

Additionally, one new issue appeared after rebase: the remote reflection is now appearing in wrong coordinates — it targets the reflection on the background, not in spotlight. I'm going to look into this soon, but maybe you can point me to right direction here too

Reply:
I think I found the issue, we are still using
```
 const targetEl = targetId
    ? (document.querySelector(`div[${DragAttribute.DROPPABLE}='${targetId}']`) as HTMLElement)
    : null
    ```
    
To determine the remote reflection coordinates

Reply:
Ah how annoying, sorry to hear that. I've checked out your branch to see I can replicate the issue, but there were some merge conflicts. After fixing them, the remote reflection appears at the top left of the screen, i.e. coordinate 0,0. 

In [this PR](https://github.com/ParabolInc/parabol/pull/5610), I pushed [a fix](https://github.com/ParabolInc/parabol/blob/23a510aff8bc9d0a396113c92738f0ffe2263c9f/packages/client/components/ReflectionGroup/ReflectionGroup.tsx#L206) to prevent remote reflections from using the position on the kanban. 

Perhaps we could Zoom and go through this bit together?

Reply:
@nickoferrall pushed a fix, can you check, please?